{"name": "test_login_negative", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('self', <SubRequest 'Setup' for <Function test_login_negative>>)", "trace": "file C:\\Users\\new ssd\\PycharmProjects\\Py2xWebAutomationFramework\\tests\\vwoLoginTests\\test_vwo_login.py, line 20\n  def test_login_negative(Setup):\nfile C:\\Users\\new ssd\\PycharmProjects\\Py2xWebAutomationFramework\\tests\\vwoLoginTests\\test_vwo_login.py, line 12\n  @pytest.fixture(autouse=True, scope='module')\n  def Setup(self):\nE       fixture 'self' not found\n>       available fixtures: Setup, _session_faker, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, extra, faker, include_metadata_in_junit_xml, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\Users\\new ssd\\PycharmProjects\\Py2xWebAutomationFramework\\tests\\vwoLoginTests\\test_vwo_login.py:12"}, "start": 1728027852599, "stop": 1728027852599, "uuid": "e29f2c7a-f5bf-47ee-8ae3-ec678f27a1ac", "historyId": "8be6628af41c999eb267bfd17245a1df", "testCaseId": "8be6628af41c999eb267bfd17245a1df", "fullName": "tests.vwoLoginTests.test_vwo_login#test_login_negative", "labels": [{"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwo_login"}, {"name": "host", "value": "DESKTOP-O5T1HPJ"}, {"name": "thread", "value": "12084-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwo_login"}]}